LayoutTests/imported/w3c/web-platform-tests/css/css-contain/container-queries/container-for-shadow-dom-expected.txt

```diff
@@ -1,12 +1,4 @@
 
-PASS Match container in outer tree
-PASS Match container in same tree, not walking flat tree ancestors
-PASS Match container in ::slotted selector's originating element tree
-PASS Match container in outer tree for :host
-PASS Match container in ::part selector's originating element tree
-PASS Match container for ::before in ::slotted selector's originating element tree
-PASS Match container in outer tree for :host::before
-PASS Match container for ::before in ::part selector's originating element tree
-PASS Match container for ::part selector's originating element tree for exportparts
-PASS Match container for slot light tree child fallback
+Harness Error (FAIL), message = ReferenceError: Can't find variable: polyfill_declarative_shadow_dom
```

Comment:
Can we add declarative-shadow-dom-polyfill.js to LayoutTests/imported/w3c/web-platform-tests/resources/ to keep this test passing?

Reply:
So currently the function is in 
https://github.com/WebKit/WebKit/blob/8ba58534414727092220b41f28de622148c21871/LayoutTests/imported/w3c/web-platform-tests/css/css-contain/container-queries/support/cq-testcommon.js#L5-L13

which is called in https://github.com/WebKit/WebKit/blob/8ba58534414727092220b41f28de622148c21871/LayoutTests/imported/w3c/web-platform-tests/css/css-contain/container-queries/container-for-shadow-dom.html#L7

@nt1m that should work no?
but indeed I can create a specific script in https://github.com/WebKit/WebKit/tree/8ba58534414727092220b41f28de622148c21871/LayoutTests/imported/w3c/web-platform-tests/resources

Why is it necessary to create it there? The WebKit tests infrastructure ?

Reply:
Just to keep in sync with upstream WPT

Reply:
@nt1m done.

Reply:
The test results don't seem up to date, the harness error is still there

Reply:
changed the expected tests. 
and added some missing ones.